Stratis.StratisD/Properties/launchSettings.json

```diff
@@ -1,7 +1,8 @@
 {
   "profiles": {
     "Stratis.StratisD": {
-      "commandName": "Project"
+      "commandName": "Project",
+      "commandLineArgs": "-testnet"
```

Comment:
This should go in the next line `"commandLineArgs": "-testnet"`

Reply:
this file should not be changed, but the PR, probably committed by mistake, will remove it